packages/workflow/src/Workflow.ts

```diff
@@ -155,6 +156,10 @@ export class Workflow {
 		this.expression = new Expression(this);
 	}
 
+	get timezone() {
+		return this.settings.timezone ?? getGlobalState().defaultTimezone;
```

Comment:
could this be set in the constructor and be marked as `readonly`, since `defaultTimezone` can't really change without a restart 🤔